### ISSUE TITLE
Accessibility and UX

### ISSUE THREAD

**Issue by entonio** [RocketChat/Rocket.Chat.Electron#879]
We've moved to Rocket.Chat from Slack and while the experience has been good there are minor accessibility points that end up being serious:
- The messages font is gray. The difference re Slack is noticeable. It's a good deal harder on my eyes.
- The general monochrome approach may be neat to others, but it's tiresome to me in extended usage. Slack has a number of coloured themes, I tend to Aubergine or Monument.
- The space above the message input field is too tall. Most of the times it's empty and just looks to me like a UI margin bug. I understand it's there for 'is typing' notifications, but why not put those below the input field where there is always space and is customary?

Sorry for mixing up issues, and on the whole kudos for a nice app, which is already in some respects superior to the others.

**Comment by tassoevan** [comment]
Issue moved to [RocketChat/Rocket.Chat #12061](https://github.com/RocketChat/Rocket.Chat/issues/12061) via [**ZenHub**](https://www.zenhub.com/)

**Comment by ekavun** [comment]
Hello! 
As a blind user, I find the accessibility of RC far superior  to Slack's. I do, however, have a frustrating issue. 
The button to access the attachment menu is simply invisible to my screenreader. I cannot upload files to a channel. I believe the button also has functions like location sharing, but I've never been able to use it so don't know the extend of what I'm missing.

Thanks!

**Comment by franckadil** [comment]
@ekavun Does your reader mentions anything about a Cross or Plus sign ? I am curious to know if it can detect glyph symbols ?

@tassoevan I think adding a label to  the icons would be a plus.

**Comment by ekavun** [comment]
Hi Franck,
On the RocketChat UI, there is no such symbol that is announced, which is somewhat the reason for the issue I'm facing. I cannot locate that Plus icon / sign to simulate, for example, a mouse click over it. 
Another issue that may be related to invisible signs or icons is that once I access any options, I cannot close the content. What I mean is that if I click on the oPtions button, or my nickname to access my account options, the content does get displayed, but I have no way of getting rid of it afterward. That is, except if I refresh the page on a chatroom, for example. So in effect, I don't have a button to close or hide the additional content I've pulled up into the page. 
I hope these issues are related and that I'm not unnecessarily polluting the thread.

Thanks!
